mono/mini/mini-runtime.c

```diff
@@ -3730,7 +3730,7 @@ mini_init (const char *filename, const char *runtime_version)
 	mono_thread_attach (domain);
 #endif
 
-	if (mono_profiler_events & MONO_PROFILE_STATISTICAL)
+	if (mono_profiler_enabled ())
```

Comment:
This is a no go change. The products install profilers during regular operations. This would cause a background thread to be created. Worse, that thread always spins, which is a power disaster.